infra/src/main/java/com/redhat/service/smartevents/infra/models/dto/UpdateManagedResourceStatusDTO.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.service.smartevents.infra.models.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class UpdateManagedResourceStatusDTO {
```

Comment:
Nice refactoring :)

I would preserve BridgeID in this class because it is a quite important contextual information for logging and error reporting. Wdyt?

Reply:
I would not add the `bridgeId` directly in this class as it would be very misleading when this class is used to update the Bridge -> what's the `id` and the `bridgeId` in this case? 

I'd rather extend this class to add all the additional metadata

Reply:
Yes in that case `id` and `bridgeId` are the same.
We can make it more explicit with a `ManagedResourceContext` class as field so that all these info (like also `customerId`)
```java
public class UpdateManagedResourceStatusDTO {

    private String id;
    private ManagedResourceContext context;
    private ManagedResourceStatus status;
}
```

Reply:
I've created a child class, as the `bridgeId` would be only specific for the `Processor`, could you take another look and gimme your feedback @danielezonca ?